LICENSE.md

```diff
@@ -0,0 +1,27 @@
+# WorkExp-CompWorkshop
+
+This work is intended for educational purposes.
+
+Copyright 2019 Science and Technology Facilities Council, U.K. Research and Innovation
+
+```
+   Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I do not think you can release this under an Apache licence if it's based on a CC BY-NC-SA licenced work:

>### Versions 2.0, 2.5, and 3.0
> 
> Your contributions to adaptations of BY-NC-SA 2.0, 2.5 or 3.0 materials may only be licensed under:
> 
>    - The license used for the original work, or a later version of that BY-NC-SA license.
>    - Ported versions of that BY-NC-SA license, the same or later version as the licensed work.

Reply:
I hadn't realised that, I'll move the License to a CC one.

Reply:
I forgot the link for my quote for reference: https://creativecommons.org/share-your-work/licensing-considerations/compatible-licenses

Reply:
Thanks - I had found this to check what needed doing. I have changed the workshop's license to a `BY-NC-SA 4.0` license, which meets the criteria of `The license used for the original work, or a later version of that BY-NC-SA license` for a 3.0 license.